README.md

```diff
@@ -26,6 +26,20 @@ app or using sbot from a cli tool.
 
 > Note: synchronous methods called over a `ssb-client` connection are converted into asynchronous methods (a callback is added as the final argument which will be called back with the result)
 
+### get (cb)
+
+get the current state of the graph. it is of the form:
+
+```
+{
+  <alice_id>: {<bob_id>:<dist>,...},...
+}
+
+```
+
+where `<dist>` is a non-zero number. (negative indicates blocking).
+a value of 1 < 1.5 is considered to mean "follows" and 0 < 0.5 is considered to mean "same as"
```

Comment:
Note that `get` is marked in `index.js` as a legacy method. Should this be moved to a Legacy section in the docs so people don't build on it?

Oh ... it is in the legacy section. Remove this?

Reply:
oh you are right. the legacy version of get returns `{<foo>:{<bar>: boolean}}` but the internal model is now `{<foo>:{<bar>: <number>}}` so really there should be a new version that returns that.